src/rebar_prv_xref.erl

```diff
@@ -37,16 +37,10 @@ init(State) ->
 -spec do(rebar_state:t()) -> {ok, rebar_state:t()} | {error, string()}.
 do(State) ->
     code:add_pathsa(rebar_state:code_paths(State, all_deps)),
-    XrefChecks = prepare(State),
-    XrefIgnores = rebar_state:get(State, xref_ignores, []),
```

Comment:
This `xref_ignores` is never added back anywhere else. Don't think we should break compatibility here.